src/test/java/org/fundacionjala/coding/maria/Kata5Test.java

```diff
@@ -0,0 +1,70 @@
+package org.fundacionjala.coding.maria;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Test to kata5.
+ */
+public class Kata5Test {
+    /**
+     * Test test01.
+     */
+    @Test
+    public void testOne() {
+        Kata5 dna = new Kata5();
```

Comment:
you are repeating this code in all the tests.
Consider adding a @Before method.